exercises/rotational-cipher/.meta/generator/rotational_cipher_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class RotationalCipherCase < Generator::ExerciseCase
+
+  def workload
+    assert_equal { "RotationalCipher.rotate(\"#{text.to_s}\", #{shiftKey.to_s})" }
```

Comment:
Thanks, `.to_s` was unnecessary. However, I can't seem to use `%Q` correctly. Calling it via `RotationalCipher.rotate(#{%Q[text]}, #{shiftKey})` generates `RotationalCipher.rotate(text, shiftKey)` instead of the value of the `text`.